### ISSUE TITLE
Add 1 additional command

### ISSUE THREAD

**Issue by willtchiu** [willtchiu/RemindMe#1]
Need a usable remind me command
- start easy (message user in the next 5 min)

**Comment by David-Egan** [comment]
So you want a function that does something after 5 minutes have passed?  It alerts the user when the 5 minutes expire?

**Comment by willtchiu** [comment]
Yup, the plan is to set up a light weight db to store user reminders. But before that, i'd like to get some working command for "remind" and delay the reminder instead of storing it.
The command would follow some sort of form: [remind 10/31/16 12:00PM PDT "Eat lunch"]

**Comment by Xh4H** [comment]
Do you still need help with it? If you do, I’d suggest leaving the task of reminding the user on his side, which is the browser (or app?)

**Comment by willtchiu** [comment]
Hey! Ah, so remove the communication between user and server? Feel free to make a PR, I've been busy and hadn't progressed with the project for a while.
